### ISSUE TITLE
authentication_oci_client.so

### ISSUE THREAD

**Issue by d-tasaki** [Sequel-Ace/Sequel-Ace#1943]
Would you bundle a plugin named authentication_oci_client.so. I would like to use SequelAce in order to connect an Oracle Cloud Managed MySQL (MDS) with an Oracle Cloud IAM user.

I tried to connect my MDS, but I got error messages as below.

> ホスト172.28.0.42に接続できないか、リクエストがタイムアウトしました。
> 
> アドレスが正しく、必要な権限があることを確認するか、接続タイムアウト（現在は10秒）を増やしてみてください。
> 
> MySQLによると：Authentication plugin 'authentication_oci_client' cannot be loaded: dlopen(/usr/local/mysql/lib/plugin/authentication_oci_client.so, 0x0002): tried: '/usr/local/mysql/lib/plugin/authentication_oci_client.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/mysql/lib/plugin/authentication_oci_client.so' (no such file), '/usr/local/mysql/lib/plugin/authentication_oci_client.so' (no such file)

**Comment by Jason-Morcos** [comment]
I don't think it would be as simple as adding this plugin - it looks like this plugin likely requires broader system access which macOS sandboxing doesn't allow. So I think it's not so simple as just bundling in that plugin - because Sequel Ace then still wouldn't be able to _access_ those federated credentials
